modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -3,18 +3,18 @@ ifndef::env-site,env-github[]
-// TODO: MULE-14438
-* If xref:execution-engine.adoc#backpressure[back-pressure] is triggered because no resources are available at the time of the trigger, that trigger is skipped. In this case a *MULE:FLOW_BACK_PRESSURE* error is thrown indicating the flow cannot accept new events at this time. In Anypoint Studio, this error is only logged once the first time it occurs. You can capture log messages each time the Scheduler is triggered, including events that do not execute the flow, by enabling debug level logging in the Mule application.  
+* If xref:execution-engine.adoc#backpressure[back pressure] occurs because Mule lacks sufficient resources to process an event in the flow, the application skips the scheduled trigger for the event and throws a MULE:FLOW_BACK_PRESSURE error. This error indicates that the flow cannot accept new events at that time. Anypoint Studio logs this error only once, the first time it occurs.
++
+Using `maxConcurrency` on a Flow component can lead to back pressure if the component contains a Scheduler that runs too frequently or if the flow takes too long to execute. To avoid back pressure and improve performance of the application, you can try the default `maxConcurrency` setting or increase the value so that more instances of the flow can run concurrently. To avoid receiving the error message, use the Scheduler's `disallowConcurrentExecution` property (see xref:scheduler-xml-reference.adoc#scheduler-properties[<scheduler> Properties]). 
++
+To capture log messages each time the Scheduler attempts to trigger an event, you can enable the xref:logging-in-mule#change-the-log-level[DEBUG] logging level in the Mule application.
 //Enhancement request for this: MULE-14930
-* If the `maxConcurrency` attribute of a Flow component is set to `1`, then only one flow instance can execute at a time in the Mule runtime. In this case, The Mule runtime prevents new flow instances from executing while the first flow instance is still executing. Instead, the Scheduler's back-pressure does *not* queue up additional events. Instead, the flow skips new events created by the triggered Scheduler while the first flow is still executing. Once the current flow terminates, the next event triggered by the Scheduler will execute a new flow instance, and then the same blocking and event skipping pattern repeats. 
-* In a multi-worker CloudHub deployment, or in a custoemr-hosted Mule runtime cluster, the Scheduler runs (and then executes the flow) only on the primary node (that is, only in one particular Mule runtime).
+* In a multi-worker CloudHub deployment or in a customer-hosted cluster of Mule runtime engines, the Scheduler runs and executes the flow on the primary node only. That is, it runs on one specific Mule runtime engine and no others.
```

Comment:
If "primary node" is well defined and applies to both CloudHub and customer-hosted cluster, ignore this question. If not, how or where or when is the Mule runtime engine that the Scheduler runs on specified?